BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c

```diff
@@ -306,6 +319,23 @@ FrameBufferWrite (
     return 0;
   }
 
+  // Handle clear screen
+  if ((NumberOfBytes == 4) && (!CompareMem (ANSI_ESCAPE_SEQ_CLEAR_SCREEN, Buffer, 4))) {
+    // Clear screen
+    SetMem (Console->TextDisplayBuf, Console->Rows * Console->Cols, 0);
+    // Zero framebuffer
+    GfxInfoHob = Console->GfxInfoHob;
+    Length = (GfxInfoHob->GraphicsMode.HorizontalResolution * GfxInfoHob->GraphicsMode.PixelsPerScanLine) * 4;
+    SetMem64 (
+        (UINT32 *) (UINTN)(GfxInfoHob->FrameBufferBase),
+        Length,
+        0
+      );
```

Comment:
For this simple function and parameter is not very long, better use single line?
SetMem64 ((UINT32 *) (UINTN)(GfxInfoHob->FrameBufferBase), Length, 0);

Reply:
Sure, will do it.

Reply:
Updated per comments.